Braintree.xcodeproj/project.pbxproj

```diff
@@ -3386,7 +3386,7 @@
 				HEADER_SEARCH_PATHS = "";
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/BraintreeCore/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
I don't think we want to change our iOS deployment target, just the Xcode version right? I didn't initially see anything in the Apple requirements indicating that iOS 14 is no longer supported, but maybe I missed it.

Reply:
Lots of versions going around so totally understandable to get lost in the version madness. 😄